testsuites/listener/shared/client_sg_topology_specific/conftest.py

```diff
@@ -9,3 +9,4 @@ def pytest_addoption(parser):
     parser.addoption("--sync-gateway-version", action="store", help="sync-gateway-version: the version of sync_gateway to run tests against")
     parser.addoption("--sync-gateway-mode", action="store", help="sync-gateway-mode: the version of sync_gateway to run tests against, channel_cache ('cc') or distributed_index ('di')")
     parser.addoption("--server-version", action="store", help="server-version: version of Couchbase Server to install and run tests against")
+    parser.addoption("--xattrs", action="store_true", help="topospecific data Use xattrs for sync meta storage. Only works with Sync Gateway 1.5.0+ and Couchbase Server 5.0+")
```

Comment:
topospecific data ?

Reply:
oops sorry, I added for debugging to differrent error from client_sg/conftest.py and client_sg_topology_specific_test/conftest.py. Thanks for spotting.

Reply:
Removed the topospecific data and checked in already and then I ran Jenkins